apps/wallet/src/ui/app/staking/stake/StakingCard.tsx

```diff
@@ -345,37 +345,38 @@ function StakingCard() {
                                 className="w-full px-0 pb-0 mx-0"
                             >
                                 <Button
-                                    size="large"
-                                    mode="neutral"
-                                    href="/stake"
+                                    size="tall"
+                                    variant="secondary"
+                                    href="#/stake"
```

Comment:
what's the purpose of the added hash fragment here and down below? do the fragments need to be a URL like `#/stake` vs `#stake`?

Reply:
@williamrobertson13 I put this in because it seemed to be breaking the app between the old version and the new version due to the `#` being missing. I'm not quite sure how the urls/routing works here given that it's a chrome extension.

Reply:
ah yeah routing in the extension works a little differently than in a traditional SPA, I think we use hash fragments in some instances and then transform the fragments into normal paths. For example, this is the URL segment from `window.location` for the account menu in the wallet 

```
/ui.html?type=popup#/tokens?menu=accounts
```